.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
```

Comment:
I think the criteria at the moment is that somebody with merge privileges has a look at it...